runtime/shared.rs

```diff
@@ -111,9 +111,9 @@ pub fn maybe_transpile_source(
     )?
     .into_source();
 
-  let maybe_source_map: Option<SourceMapData> = transpiled_source
-    .source_map
-    .map(|sm| sm.into_bytes().into());
+  let maybe_source_map: Option<SourceMapData> =
+    transpiled_source.source_map.map(|sm| sm.into());
+  let source_text = String::from_utf8(transpiled_source.source)?;
```

Comment:
We could avoid this `String::from_utf8` by not requiring a FastString in deno_core.

Reply:
This seems fine, because it's only used during build time and we want to use the fastest available "string type" for the snapshotted sources.